toy.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from skglm.datafits import Quadratic
+from skglm.penalties import SCAD
+from skglm.solvers.cd_solver import cd_solver
+
+
+n_samples = 1000
+n_features = 100_000
+
```

Comment:
After @Klopfe 's fix of prox computation, this toy script does converge in small and large settings, for arbitrarily small tolerance. If ok, I can remove it.

Reply:
ok for me